engine/src/cmd/collide2/CSopcodecollider.cpp

```diff
@@ -128,7 +128,7 @@ csOPCODECollider::~csOPCODECollider ()
 }
 
 
-void csOPCODECollider::MeshCallback (udword triangle_index,
+void csOPCODECollider::MeshCallback (ice_udword triangle_index,
```

Comment:
Most of the changes in this PR are similar to this one, where I just renamed some integer type aliases used by `ICE` and `opcode` or `OPC`. Didn't want them to conflict with each other's (or anyone else's) typedefs.

Reply:
Update: No longer applicable